src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -814,27 +814,23 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
     /// Let $x$ be the input vector, $n$ the size of the vector, $L(x)$ the norm function selected by the user.
     /// Let $\mu(x) = \sum_i x_i / n$ be the mean of the values of vector $x$. The <xref:Microsoft.ML.Transforms.LpNormNormalizingTransformer>
     /// performs the following operation on each input vector $x$:
-    ///
-    ///     $y = \frac{x - \mu(x)}{L(x)}$
-    ///
+    /// $y = \frac{x - \mu(x)}{L(x)}$
     /// if the user specifies that the mean should be zero, or otherwise:
-    ///
-    ///     $y = \frac{x}{L(x)}$
-    ///
+    /// $y = \frac{x}{L(x)}$
     /// There are four types of norm that can be selected by the user to be applied on input vector $x$. They are defined as follows:
-    /// - <xref:Microsoft.ML.Transforms.LpNormNormalizingEstimatorBase.NormFunction.L1>
+    /// - <xref:Microsoft.ML.Transforms.LpNormNormalizingEstimatorBase.NormFunction.L1>:
     ///
     ///     $L_1(x) = \sum_i |x_i|$
```

Comment:
> $L_1(x) = \sum_i |x_i|$ [](start = 11, length = 24)

Let's move these righ after ':' on the same line. they'd look nicer: 
[https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.lpnormnormalizingestimator?view=ml-dotnet&branch=smoke-test-preview](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.lpnormnormalizingestimator?view=ml-dotnet&branch=smoke-test-preview)